docs/admin/install/operator/configuring-eventing-cr.md

```diff
@@ -316,3 +314,100 @@ spec:
       cpu: 1000m
       memory: 250Mi
 ```
+
+### Override the nodeSelector
```

Comment:
I don't know about English grammar :sweat_smile: but should we remove `the` in `Override the nodeSelector` or add it in `Override tolerations` below?